### ISSUE TITLE
exec on `testRpc` child process will kill child if stdout buffer overflows

### ISSUE THREAD

**Issue by rudolfix** [sc-forks/solidity-coverage#120]
```
this.testrpcProcess = childprocess.exec(command + options, null, (err, stdout, stderr) => {
```
Maximum size of stdout buffer is 200KB by default. Child process is killed if such size is exceeded. It's not too hard, testrpc is printing a lot by default
https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

Solution is increasing this buffer to much larger size or force silent mode on testRpc (however you use stdout to check if testrpc started correctly). Could be similar to this:
https://github.com/Neufund/solidity-coverage/commit/08bf24f748d407da88e718edda6a859d05250f91

**Comment by cgewecke** [comment]
@rudolfix - Good to know - that's interesting. Thanks for the fix too, much appreciated.

**Comment by cgewecke** [comment]
Resolved with ed1490f609004e7be9a943dc201be0c81d6a728a.

**Comment by cgewecke** [comment]
Published: `0.2.6`
